src/main/markdown/faq.md

```diff
@@ -0,0 +1,49 @@
+
+## Does Lagom require me to use Cassandra for persistence?
+
+No, Lagom supports using either Cassandra or relational databases, including MySQL, PostegreSQL, Oracle and H2. The Lagom development environment provides and uses Cassandra by default because it offers great scalability and replication features.  See more in the [Java documentation](/documentation/current/java/PersistentEntity.html) or the [Scala documentation](/documentation/current/scala/PersistentEntity.html).
+
+## Does Lagom require a me to run a Kafka cluster?
+
+No, but it is very useful for common communication scenarios. Lagom's Message Broker API integrates Persistent Entities with [Apache Kafka](https://kafka.apache.org/) to provide publish and subscribe messaging between services with at-least-once delivery guarantees. This enables you to design decoupled and highly-resilient systems of microservices.
+
+If you prefer not to run a Kafka cluster, however, the Message Broker API is completely optional. You can write Lagom services that communicate with each other directly by using HTTP requests or by streaming data over a WebSocket.
+
+The Message Broker API was designed to make it possible to support for alternative message broker services in a future version of Lagom.
+
+See more in the [Java documentation](/documentation/current/java/MessageBroker.html) or the [Scala documentation](/documentation/current/scala/MessageBroker.html).
+
+## Does Lagom require using event sourcing and CQRS?
+
+You can handle data persistence any way you like in a Lagom service, but we strongly encourage event sourcing and CQRS, and the Lagom Persistence API is designed to support these patterns. For services where you do not wish to use event sourcing and CQRS, you can execute arbitrary database statements using Lagom's `CassandraSession`, `JdbcSession` or `JpaSession`. You can also use any other data access client you choose, as long as you are careful to avoid I/O blocking the primary thread pool.
+
+See more in the Lagom documentation on [Managing data persistence](/documentation/current/java/ES_CQRS.html) and [Advantages of Event Sourcing](/documentation/current/java/ESAdvantage.html).
+
+## Can I serialization methods other than JSON?
+
+Yes, Lagom supports extensible serialization, both for messages transmitted between different services and for internal messages and persistent data used within a service. JSON is supported by default, as it is simple and well understood. Services that require higher performance or more compact data can use alternatives such as [Protocol Buffers](https://developers.google.com/protocol-buffers/), [Apache Avro](http://avro.apache.org/), [Kryo](https://github.com/EsotericSoftware/kryo), or any other serializer that you choose.
```

Comment:
I'd love to have examples of theses.

And even https://capnproto.org/

Reply:
I was thinking the same thing... good ideas for the recipes repo.